go-controller/pkg/ovn/zone_interconnect/zone_ic_handler.go

```diff
@@ -617,7 +626,11 @@ func (zic *ZoneInterconnectHandler) addRemoteNodeStaticRoutes(node *corev1.Node,
 			var err1 error
 			nodeGRPIPs, err1 = util.ParseNodeGatewayRouterLRPAddrs(node)
 			if err1 != nil {
-				return fmt.Errorf("failed to parse node %s Gateway router LRP Addrs annotation %w", node.Name, err1)
+				err1 = fmt.Errorf("failed to parse node %s Gateway router LRP Addrs annotation %w", node.Name, err1)
```

Comment:
hmm why are we reconstructing the err again and parsing that.. ? for more info?

Reply:
yeah i just left the message intact and only changed suppression

Reply:
done!